cocoa/dlg.m

```diff
@@ -52,6 +52,11 @@ - (DlgResult)run {
 		[alert addButtonWithTitle:@"OK"];
 		break;
 	}
+	[NSApp setActivationPolicy:NSApplicationActivationPolicyRegular];
```

Comment:
Judging by the [docs](https://developer.apple.com/documentation/appkit/nsapplicationactivationpolicy?language=objc), the type of command-line usage you are talking about should probably use `NSApplicationActivationPolicyAccessory`?

This is the kind of application-level decision that I don't think dialog really has any business fiddling with, however I would be ok with this being done conditionally -- ie. if `[NSApp activationPolicy]` is `NSApplicationActivationPolicyProhibited` then it's reasonable for dialog to change it in the interest of ease-of-use, but if it is already set to another value then we shouldn't touch it.